react/Tile/styles.styl

```diff
@@ -43,3 +43,70 @@ $tileMobileHeight = 3.75rem
   .Tile:active,
   .Tile:focus
     transform scale(1.01)
+
+.Tile-icon-wrapper
+  margin-top .25rem
+  margin-bottom .25rem
+  width 3rem
+  height 3rem
+
+.Tile-desc
+  display flex
+  flex-direction column
+  justify-content flex-start
+  align-items center
+  margin-top .5rem
+  text-align center
+  max-width 100%
+
+.Tile-title, .Tile-developer, .Tile-status
+  display block
+  margin 0
+  text-overflow ellipsis
+  white-space nowrap
+  overflow hidden
+  max-width 100%
+
+.Tile-title
+  height 1.25rem
+  color var(--black)
+
+.Tile-developer
+  height 1rem
+
+.Tile-status
+  margin-top .5rem
+  height 1rem
+  color var(--coolGrey)
+
+
++small-screen()
+  .Tile-icon-wrapper
+    margin-bottom 0
+
+  .Tile-icon
+    height 2.5rem
+    max-width 2.5rem
+    max-height 2.5rem
+
+  .Tile-desc
+    margin-top .125rem
+    margin-left .5rem
+    justify-content center
+    text-align left
+    /* 4rem for icon and margins */
+    width calc(100% - 4rem)
+
+  .Tile-title
+    height 1rem
+    font-weight normal
+    font-size 12px
```

Comment:
shouldn't we use 'rem' instead of 'px' ?

Reply:
This file was only moved, I did not touch its content, but you're right that it's better to use rem as a general rule. Even better, we have typography variables, I made the change in my last commit, thanks.

https://github.com/cozy/cozy-ui/pull/1560/commits/2c638b18512761a82e5fbfdc9d9c608983165fd5